spec/requests/api/v1/documents_spec.rb

```diff
@@ -3,6 +3,13 @@
 
 describe API::V1::Documents do
   let(:id) { "some really/weird@id.name" }
+  let(:valid_params) do
+    {
+     "document_id" => id, "title" => "my title", "path" => "http://www.gov.gov/goo.html",
```

Comment:
Can do. For some reason I thought this was correct, but the style guide agrees with you, so...